src/collective/behavior/amp/browser/view.pt

```diff
@@ -217,6 +218,8 @@
         tal:content="structure sticky_ad">
     </amp-sticky-ad>
 
+    <amp-pixel tal:condition="pixel" tal:replace="structure pixel" />
```

Comment:
in fact I think I can remove the `tal:condition` here as the method will return `None` in case no adapter is registered and that will remove the tag.